examples/apps/cchmc_ped_abd_ct_seg_app/dicom_seg_writer_operator.py

```
@@ -0,0 +1,529 @@
+            if Path(seg_image_file).is_file():
+                seg_image = self._image_file_to_numpy(seg_image_file)
+            else:
+                raise ValueError("Input 'seg_image' is not an Image or a path.")
+
+        # If the optional named input, output_folder, has content, use it instead of the one set on the object.
+        # Since this input is optional, must check if data present and if Path or str.
+        output_folder = None
+        try:
+            output_folder = op_input.receive(self.input_name_output_folder)
+        except Exception:
+            pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)